examples/solids/index.rst

```diff
@@ -0,0 +1,348 @@
+
+.. math::
+
+   \mathsf C = \dfrac{E}{(1+\nu)(1-2\nu)}
+   \begin{bmatrix}
+        1-\nu & \nu & \nu & & & \\
+          \nu & 1 - \nu & \nu & & & \\
+          \nu & \nu &  1 - \nu & & & \\
+          & & & \dfrac{1 - 2\nu}{2} & & \\    
+         & & & &\dfrac{1 - 2\nu}{2} & \\
+         & & & & & \dfrac{1 - 2\nu}{2} \\   
+   \end{bmatrix},
+
+where :math:`E` is the Young’s modulus and :math:`\nu` is the Poisson’s ratio.
+An alternate formulation is in terms of the Lamé parameters,
+
+.. math::
+   \begin{aligned}
+   \lambda &= \frac{E \nu}{(1 + \nu)(1 - 2 \nu)} \\
+   \mu &= \frac{E}{2(1 + \nu)}
+   \end{aligned}
+
+in which case the constitutive equation :math:numref:`linear-stress-strain` may be written
+
+.. math::
+   \bm\sigma = \lambda (\operatorname{trace} \bm\epsilon) \bm I_3 + 2 \mu \bm\epsilon,
+
+where :math:`\bm I_3` is the :math:`3 \times 3` identity matrix.
+Alternatively, the stiffness tensor becomes
+
+.. math::
+
+   \mathsf C = \begin{bmatrix}
+   \lambda + 2\mu & \lambda & \lambda & & & \\
+   \lambda & \lambda + 2\mu & \lambda & & & \\
+   \lambda & \lambda & \lambda + 2\mu & & & \\
+   & & & \mu & & \\
+   & & & & \mu & \\
+   & & & & & \mu
+   \end{bmatrix}.
+
+Note that the incompressible limit :math:`\nu \to \frac 1 2` causes :math:`\lambda \to \infty`, and thus :math:`\mathsf C` becomes singular.
+
+.. _problem-hyper-small-strain:
+
+Hyperelasticity at Small Strain
+----------------------------------------
+
+The strong form of the static balance linear momentum at small strain for the three-dimensional neo-Hookean Hyperelasticity problem is given by
+:cite:`hughes2012finite`:
+
+.. math::
+
+   \nabla \cdot \boldsymbol{\sigma} + \boldsymbol{g} = \boldsymbol{0} 
+
+Integrating by parts on the divergence term, we arrive at the weak form:
+
+.. math::
+
+   \int_{\Omega}{ \nabla \boldsymbol{v} \colon \boldsymbol{\sigma}} dV - \int_{d\Omega}{\boldsymbol{v} \cdot \left(\boldsymbol{\sigma}_t \cdot \hat{\boldsymbol{n}}\right)} dS + \int_{\Omega}{\boldsymbol{v} \cdot \boldsymbol{g}} dV = 0
+
+where :math:`\boldsymbol{\sigma}_t \cdot \hat{\boldsymbol{n}}` is typically replaced with a boundary condition.
+
+The small strain version of a Neo-Hookean hyperelasticity material is given as
+follows:
+
+.. math::
+   :label: clss
+   
+   \boldsymbol{\sigma} = \lambda \ln(1 + \boldsymbol{\epsilon_v)} \boldsymbol{I}_3 + 2\mu \boldsymbol{\epsilon}
+
+where :math:`\boldsymbol{\sigma}`, :math:`\boldsymbol{\epsilon}`, are stress and
+strain, respectively. Strain is defined by
+
+.. math::
+
+   \boldsymbol{\epsilon} = \dfrac{1}{2}\left(\nabla \boldsymbol{u} + \nabla \boldsymbol{u}^T \right)
+
+where :math:`\bm u` is the displacement and :math:`\mu` and  :math:`\lambda` are the Lamé parameters.
```

Comment:
Sorry, I deleted the comment and retyped it here, because I had accidentally dragged a wrong initial line. Here it is :
I would remove these lines too. The symbols and nomenclature have not changed since these symbols have been introduced above.